decidim-meetings/config/locales/en.yml

```diff
@@ -74,5 +74,7 @@ en:
             start_time: Start date
             title: Title
     resource_links:
+      meetings_from_result:
+        results: 'Meeting appearing in these results:'
```

Comment:
Design has "Propuestas tratadas" as title of the section (but it's actually showing results). What do you suggest?